### PR TITLE
Gtk: Fix support for Gtk packages included with Stretch

### PR DESCRIPTION
**Pull request by tombettany**
The upgrade to Stretch brings a new version of Gtk which breaks some of
the CSS. Resolve these issues.

### CODE REVIEW

File: media/css/style.css [hunk 1]
```diff
@@ -53,25 +53,29 @@ GtkScrolledWindow, GtkViewport {
...
-    font: Bariol Bold 16;
+    font-family: "Bariol Bold";
+    font-size: 16pt;
     border-radius: 3px;
 }
 
 .desktop_toggle_button {
     background-color: #91D064;
-    font: Bariol Bold 16;
+    font-family: "Bariol Bold";
```
Comment (radujipa) [inline media/css/style.css]: Good find with this monster

Comment (wizofe) [reply media/css/style.css]: Oh so do we change all of the instances to Bariol Bold? The other solution would be to find the Bariol font with embedded weights.

Comment (tombettany) [reply media/css/style.css]: I'll take a look at this whole bold issue - certainly the easiest thing right now is to just declare it as `Bariol Bold` but @wizofe you are right, we should look to do this properly.

Comment (tombettany) [reply media/css/style.css]: Given that Gtk is completely ignoring `font-weight`, I'll merge this as-is and make some alterations elsewhere to use `Bariol Bold` over `font-weight: bold`.

**Pull request merged**